src/main/java/uk/gov/cshr/service/InviteService.java

```diff
@@ -13,29 +13,24 @@
 import uk.gov.cshr.domain.InviteStatus;
 import uk.gov.cshr.domain.Role;
 import uk.gov.cshr.repository.InviteRepository;
-import uk.gov.service.notify.NotificationClient;
 import uk.gov.service.notify.NotificationClientException;
-import uk.gov.service.notify.SendEmailResponse;
 
 import java.util.Date;
-import java.util.HashMap;
 import java.util.Set;
 
 @Service
 @Transactional
 public class InviteService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(InviteService.class);
-    private static final String EMAIL_PERMISSION = "email";
-    private static final String ACTIVATION_URL_PERMISSION = "activationUrl";
 
-    private InviteRepository inviteRepository;
+    @Autowired
+    private NotifyService notifyService;
 
-    @Value("${govNotify.key}")
-    private String api;
+    private InviteRepository inviteRepository;
 
-    @Value("${govNotify.template}")
-    private String templateId;
+    @Value("${govNotify.inviteTemplate}")
```

Comment:
We will have more than one template, let's group them under `govNofity.template.invite` structure

Reply:
Done